paper-inky-focus-behavior.html

```diff
@@ -13,8 +13,7 @@
 
 <script>
 
-  /** @polymerBehavior */
-  Polymer.PaperRadioButtonInk = {
+  Polymer.PaperInkyFocusBehaviorImpl = {
```

Comment:
I think that the way we are supposed to annotate this is by adding:

```javascript
/** @polymerBehavior Polymer.PaperInkyFocusBehavior */
```

If you add it here _and_ above the Array down at the bottom, the doc viewer is supposed to use both thingies.